### DISCUSSION TITLE
How to return unique field values from a table's string array field?

### DISCUSSION THREAD

**Discussion by AndrewGalea** [prisma/prisma#17256] [Q&A]
I have the following MongoDB table schema:

`faqs {
      id: String
      answer: String
      question: String
      tags: String[]
}`

I want to return a unique list of all `tags` in the table. Is there a way to do this with Prisma? I assume it has to be via the `$queryRaw` method; however, I have been unable to get this to work. I would appreciate any help.

**Comment by nurul3101** [comment]
Hello @AndrewGalea 👋 

To confirm, do you mean you want to get all the unique tags in a single document or all combined unique tags for all the documents in the model?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
